packages/test/test-end-to-end-tests/src/test/gc/gcTombstone.spec.ts

```diff
@@ -564,6 +564,36 @@ describeNoCompat("GC tombstone tests", (getTestObjectProvider) => {
             sendDataObject._runtime.submitSignal("can receive", "a signal");
             await provider.ensureSynchronized();
         });
+
+        itExpects("does not throw tombstone errors with ThrowOnTombstoneUsage setting not enabled",
```

Comment:
We need to also change `datastores.ts` as we also need to be able to request the tombstoned datastore.

Reply:
Fixed it.